packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -180,6 +180,16 @@ public void showEditorHelp() {
         emitToJS(EVENT_NAME_SHOW_EDITOR_HELP, null);
     }
 
+    @ReactMethod
+    public void addListener(String eventName) {
+        // Keep: Required for RN built in Event Emitter Calls.
+    }
+
+    @ReactMethod
+    public void removeListeners(Integer count) {
+        // Keep: Required for RN built in Event Emitter Calls.
+    }
+
```

Comment:
I'm really surprised that RN is set up so that failing to manually add these empty methods on Android results in a _runtime_ error, but it does appear to be the way to address this. Just adding a couple of links I found useful when digging into this a bit.

https://github.com/facebook/react-native/commit/114be1d2170bae2d29da749c07b45acf931e51e2
https://github.com/facebook/react-native-website/pull/2791#discussion_r721584017